azure-pipelines.yml

```diff
@@ -121,11 +134,11 @@ jobs:
 
     strategy:
       matrix:
-        mac-py36:
-          image_name: macos-10.13
-          test_title: macOS py36
-          toxenv: py36
-          python: 3.6
+        # mac-py36:
+        #   image_name: macos-10.13
+        #   test_title: macOS py36
+        #   toxenv: py36
+        #   python: 3.6
```

Comment:
Why was this done? Anticipating removing support for python 3.6?

Reply:
I actually just didn't want to deal with the `dataclasses` dependency at this point. We call pull `dataclasses` back in as a dependency if python==3.6 if we like. Or we can remove `dataclasses` from the code base if we don't need them. We never did actually use them in the end.

Either options will allow us to support Python 3.6 again...